api/models/User.js

```diff
@@ -34,14 +34,14 @@ module.exports = {
     id: {
       type: 'string',
       primaryKey: true,
-      unique: true,
-      index: true,
       uuidv4: true,
-      defaultsTo: function (){ return uuid.v4(); }
+      defaultsTo() {
+        return uuid.v4();
+      }
     },
     credit: {
-      type: 'string',
-      defaultsTo: '0'
+      type: 'integer',
```

Comment:
I am not sure this will work. See https://github.com/Nanocloud/nanocloud/blob/b8e07708813596ba84326b5e9c8aac8fe24bed7c/api/services/MachineService.js#L148